packages/redux-dynamic-modules-observable/README.md

```diff
@@ -1 +1,41 @@
-#This is still a WIP. Contributions are welcome
+
+## Usage with redux-observable
+
+You can use `redux-dynamic-modules` alongside `redux-observable` so that you can add/remove Epics along with your modules.
+
+To use
+
+-   `npm install redux-dynamic-modules-observable`
+-   Add the observable extension to the `createStore` call
+
+```typescript
+import { createStore, IModuleStore } from "redux-dynamic-modules";
+import { getObservableExtension } from "redux-dynamic-modules-observable";
+import { getUsersModule } from "./usersModule";
+
+const store: IModuleStore<IState> = createStore(
+    /* initial state */
+    {},
```

Comment:
We should update the other examples, but can you update this to use the new `createStore` object argument?

Reply:
I updated `readme.md` file. You may check and review it later.